src/Arbitrary.hs

```diff
@@ -385,7 +385,7 @@ instance Arbitrary Timestamp where
     arbitrary = Timestamp <$> arb
 
 instance GenArbitrary r => GenArbitrary (Action r) where
-    genArbitrary = liftIO $ generate arbitrary
+    genGen = liftIO . generate
```

Comment:
I may miss a point here, but I think I disagree.  I am in favour of calling things different if they are technically different.  `generate` requires `IO`, and we want to get away from `IO` here.

I'm not fond of the `genGen` name, but I would like to keep a name that's not `generate`, and avoid the default method trick.